### PR TITLE
modifty kout request param order

### PR DESCRIPTION
**Pull request by shzcore**

### CODE REVIEW

**Review by javeme — DISMISSED** [review]

**Review by Linary — DISMISSED** [review]

**Comment by javeme** [comment]
CI fails:
image

File: src/test/java/com/baidu/hugegraph/api/traverser/KoutApiTest.java [hunk 1]
```diff
@@ -113,7 +113,7 @@ public void testKoutGetBothAllWithCapacity() {
                         2, false, -1L, -1L, 1L);
         }, e -> {
             String expect = "Capacity can't be less than limit, " +
-                    "but got capacity '1' and limit '-1'";
+                    "but got capacity '-1' and limit '1'";
```
Comment (javeme) [inline src/test/java/com/baidu/hugegraph/api/traverser/KoutApiTest.java:116]: align with "Capacity..."

Comment (javeme) [reply src/test/java/com/baidu/hugegraph/api/traverser/KoutApiTest.java:116]: @zhoney does capacity=-1 means capacity > limit 1

**Review by javeme — CHANGES_REQUESTED** [review]

File: src/test/java/com/baidu/hugegraph/api/traverser/KoutApiTest.java [hunk 2]
```diff
@@ -110,10 +110,10 @@ public void testKoutGetBothAllWithCapacity() {
...
             koutAPI.get(markoId, Direction.BOTH, null,
-                        2, false, -1L, -1L, 1L);
+                        2, false, -1L, 1L, 2L);
         }, e -> {
             String expect = "Capacity can't be less than limit, " +
-                    "but got capacity '1' and limit '-1'";
+                    "but got capacity '1' and limit '2'";
             Assert.assertContains(expect, e.getMessage());
         });
     }
```
Comment (zhoney) [inline src/test/java/com/baidu/hugegraph/api/traverser/KoutApiTest.java:119]: add one test with capacity 1 and limit -1 to ensure limit can't be NO_LIMIT when capacity is not NO_LIMIT

File: src/main/java/com/baidu/hugegraph/api/traverser/KoutAPI.java [hunk 3]
```diff
@@ -43,7 +43,7 @@ protected String type() {
```
Comment (zhoney) [inline src/main/java/com/baidu/hugegraph/api/traverser/KoutAPI.java:43]: rebase #113

**Review by zhoney — CHANGES_REQUESTED** [review]

**Review by javeme — APPROVED** [review]

**Review by zhoney — APPROVED** [review]

**Pull request merged**